Library/include/CSP/Systems/Users/UserSystem.h

```diff
@@ -62,10 +62,12 @@ class CSP_API CSP_NO_DISPOSE UserSystem : public SystemBase
 	/// @param UserName csp::common::String
 	/// @param Email csp::common::String
 	/// @param Password csp::common::String
+	/// @param UserHasVerifiedAge csp::common::Optional<bool> : An optional bool to specify whether or not the user has verified that they are over 18
 	/// @param Callback LoginStateResultCallback : callback to call when a response is received
 	CSP_ASYNC_RESULT void Login(const csp::common::String& UserName,
 								const csp::common::String& Email,
 								const csp::common::String& Password,
+								const csp::common::Optional<bool>& UserHasVerifiedAge,
```

Comment:
Considering the age should be explicitly 18 and over, it would be nice to have it captured it as part of the variable name.

Reply:
If that's the case, perhaps we could just match the comment with the variable name ? As the comment suggests 18

Reply:
I deliberately chose not mention a specific age here, as the plan is for us to eventually allow 13+.

Reply:
Unless we feel strongly otherwise, let's keep it general for now as it does give us latitude in the future.

Reply:
Happy with either to be honest, up to you. One small note for specificity would be that the CHS endpoint variable name is verifiedAgeEighteen based on their integration doc. This value aims to wrap that in a sense, so it would make sense to match its name instead of generalising it. Either way, it feels like the xml comment should match the value on whether '18>' is mentioned or not.

Reply:
Agreed on that last part.